[2023-12-21 18:00:06,384] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: dag_with_python_weather_v01.fill 2023-12-21T17:00:00+00:00 [queued]>
[2023-12-21 18:00:06,568] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: dag_with_python_weather_v01.fill 2023-12-21T17:00:00+00:00 [queued]>
[2023-12-21 18:00:06,568] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2023-12-21 18:00:06,569] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2023-12-21 18:00:06,569] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2023-12-21 18:00:06,599] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): fill> on 2023-12-21T17:00:00+00:00
[2023-12-21 18:00:06,639] {standard_task_runner.py:52} INFO - Started process 15163 to run task
[2023-12-21 18:00:06,706] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'dag_with_python_weather_v01', 'fill', '2023-12-21T17:00:00+00:00', '--job-id', '228', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_python_weather.py', '--cfg-path', '/tmp/tmp01_jlwci', '--error-file', '/tmp/tmpe4fp37iw']
[2023-12-21 18:00:06,731] {standard_task_runner.py:77} INFO - Job 228: Subtask fill
[2023-12-21 18:00:07,052] {logging_mixin.py:104} INFO - Running <TaskInstance: dag_with_python_weather_v01.fill 2023-12-21T17:00:00+00:00 [running]> on host 5f3722f1fc3b
[2023-12-21 18:00:07,375] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=ggoni
AIRFLOW_CTX_DAG_ID=dag_with_python_weather_v01
AIRFLOW_CTX_TASK_ID=fill
AIRFLOW_CTX_EXECUTION_DATE=2023-12-21T17:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-12-21T17:00:00+00:00
[2023-12-21 18:00:09,353] {python.py:118} INFO - Done. Returned value was: None
[2023-12-21 18:00:09,404] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=dag_with_python_weather_v01, task_id=fill, execution_date=20231221T170000, start_date=20231221T180006, end_date=20231221T180009
[2023-12-21 18:00:09,475] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-12-21 18:00:09,501] {local_task_job.py:146} INFO - Task exited with return code 0
